model/flow/sealing_segment.go

```diff
@@ -48,23 +50,29 @@ type SealingSegmentBuilder struct {
 
 // AddBlock appends block to blocks
 func (builder *SealingSegmentBuilder) AddBlock(block *Block) error {
-	for _, receipt := range block.Payload.Receipts {
-		if _, ok := builder.includedResults[receipt.ResultID]; ok {
-			continue
-		}
+	//sanity check: block should be 1 height higher than current highest
+	if !builder.isValidHeight(block) {
+		return fmt.Errorf("invalid block height (%x): %w", block.Header.Height, ErrSegmentInvalidBlockHeight)
```

Comment:
```suggestion
		return fmt.Errorf("invalid block height (%d): %w", block.Header.Height, ErrSegmentInvalidBlockHeight)
```

I don't the we want the height hex-formatted